playbooks/molecule/resources/xnat/inventory/hosts.yml

```diff
@@ -9,7 +9,7 @@ all:
       ansible_port: 22
     # Host for your web server. Variables in host_vars/xnat_web will be available to this host
     xnat_web:
-      hostname: "xnat.web.local"
+      hostname: "localhost" # necessary to allow redirects outside the Docker network and to avoid CORS issues inside the network
```

Comment:
Oh, how the tables have turned.

Investigative work carried out by @HChughtai & @paddyroddy on https://github.com/UCL-MIRSG/UCLH-MPBE-SRR-XNAT